views/items/index.pug

```diff
@@ -15,6 +15,11 @@ mixin display( nodes, depth )
         +display( entry.children, depth + 1 )
 
 block content
+
+  #myModal.modal.fade(tabindex='-1', role='dialog', aria-labelledby='myModalLabel', aria-hidden='true')
```

Comment:
@jrob i changed the id name and the modal is showing up but i dont know if it is rendering the correct html content